src/components/counter.component.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+
+class Counter extends Component {
+  state = { count: 0 }
+
+  handleIncrease = () => {
+    this.setState({ count: this.state.count + 1 })
+  }
+
+  handleDecrease = () => {
+    this.setState({ count: this.state.count - 1 })
+  }
+  handleSubmit = event => {
+    event.preventDefault()
+    let input = event.target[0].value
+    this.setState({ count: parseInt(input, 10) })
+  }
+
+  render() {
+    const buttonStyles = {
+      border: '1px solid #ccc',
+      background: '#fff',
+      fontSize: '2em',
+      padding: 15,
+      margin: 5,
+      width: 200
+    }
```

Comment:
styling should be done in the `.scss` files. the only time you should use the `style` prop is if the style attribute is being calculated with javascript